Documentation/README.md

```diff
@@ -112,8 +112,8 @@ In addition File has the following nodes:
 
  Attribute | Optional | Description |
 :-----------|:------------|:------------
-Filename | Yes | The BIM file related to this topic.
-Date | Yes | Date of the BIM file.
+Filename | Yes | The BIM file related to this topic. For IFC files this is the first item in the FILE_NAME entry in the IFC file's [header](https://standards.buildingsmart.org/documents/Implementation/ImplementationGuide_IFCHeaderData_Version_1.0.2.pdf).
```

Comment:
Should we maybe add something like _This is not the actual filename as saved to disk, since that can change at any time._?

Reply:
I vote to keep Pasi's suggestion as-is. Its short and clear and refers people to a clarifying guide.